### DISCUSSION TITLE
Support for Deployer 7 yml files

### DISCUSSION THREAD

**Discussion by danshumaker** [sourcebroker/deployer-extended-wordpress#12] [General]
Would it be possible to use https://github.com/deployphp/deployer/releases/tag/v7.0.0-beta.24  with this?

Looks like from the composer.json file that 6.8 is the latest version possible.  Yaml config files would be nice to use.

**Comment by kszymukowicz** [comment]
The answer is the same as here:
https://github.com/sourcebroker/deployer-extended-database/discussions/12
